src/components/StyleGuide/Filtering/PageCompositionElements/FilterBar.mdx

```diff
@@ -0,0 +1,232 @@
+import { Meta } from "@storybook/blocks";
+
+import { Box } from "@chakra-ui/react";
+import Banner from "../../../Banner/Banner";
+import Image from "../../../Image/Image";
+import Link from "../../../Link/Link";
+
+import imageExampleColumn from "/static/filteringStyleGuide/exampleColumn.png";
+import imageExamplePopup from "/static/filteringStyleGuide/examplePopup.png";
+import imageExampleRow from "/static/filteringStyleGuide/exampleRow.png";
+import imageFilterBar from "/static/filteringStyleGuide/filterBar.png";
+import { exampleWrapperStyles } from "../Overview.mdx";
+
+<Meta title="Style Guide/Filtering/Page elements/Filter bar" />
+
+# Filter bar
+
+The `filter bar` is a wrapper for individual form input elements, including the
+following DS components: `CheckboxGroup`, `MultiSelectGroup`, `RadioGroup`,
```

Comment:
I think this gives the impression that we are supporting managing state for all components but right now we only manage state for `MultiSelect`.

Reply:
What is giving that impression? Calling it a `wrapper`?